docs/readthedocs/source/doc/Nano/Howto/Training/TensorFlow/tensorflow_training_embedding_sparseadam.nblink

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Embedding layers are often used to encode categorical items in deep learning applications. However, in applications such as recommendation systems, the embedding size may become huge due to large number of items or users, leading to extensive costs in both space and computation. 

Sparse Embedding is a common way to reduce the cost of very large embeddings. BigDL-Nano provides the Embedding layer class (with sparse support) and SparseAdam optimizer to optimize model with large embeddings in TensorFlow Keras applications.

Reply:
Have revised.